application/controllers/handler.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
       registered: (new Date()).toISOString(),
       providers: []
     };
-    console.log('Registration: ', user);
+    console.log('Registration: ', user.username, user.email);
```

Comment:
Is this enough information or would it be preferable to just strip the secret information by doing `delete user.XYZ;`